docs/guides/documentation/releases/README.md

```diff
@@ -0,0 +1,94 @@
+# Releases
+
+<pre align="center">Streamline software releases through consistent templates, automated aggregation and notifications.</pre>
+
+## Introduction
+
+**Background**: Making releases can be a chore. From ensuring the release notes have helpful information to aggregating many updates and notifying your users - doing the process right can be time consuming and fraught with misunderstandings. In this guide we recommend a standardized, automated way to notify your users of new releases. For example, we recommend using a `.github/release.yml` template to allow for more readable release notes, showcasing the importance of each update. We also provide automation for aggregating releases from many repositories into a super-release that provides a holistic view of project progress. Finally, we recommend automated notifications through GitHub integrations to help keep everyone informed.
```

Comment:
Great intro! As a personal aside, I prefer `.yaml` to `.yml`—unless you also name your HTML files `.htm` 😉

Reply:
Haha - thanks @nutjob4life - `.htm` brings back repressed memories of Microsoft Frontpage days 😆 .

Side note though - not sure if `.yaml` would work with GH's system: https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes#configuring-automatically-generated-release-notes

Reply:
Dark days, those Frontpage days 😝

Reply:
Sometimes in place of an emdash I use double hyphens which some markup translates to an emdash.